decidim-comments/app/frontend/comments/comments.component.jsx

```diff
@@ -21,8 +21,21 @@ import commentsQuery            from './comments.query.graphql';
  */
 export class Comments extends Component {
   render() {
-    const { comments, reorderComments, orderBy } = this.props;
+    const { comments, reorderComments, orderBy, loading } = this.props;
 
+    if (loading) {
+      return (
+        <div className="columns large-9" id="comments">
+          <section className="comments loading-comments">
+            <div className="row collapse order-by">
+              <h2 className="order-by__text section-heading">
+                { I18n.t("components.comments.loading") }
```

Comment:
I don't think duplicating the whole markup is a good idea. What about creating a `_renderHeader()` function instead that conditionally renders one text or another?

If you want to add an extra `loading-comments` class, you can always: `className={ commentClasses }` and specify it as a variable above:

```javascript
let commentClasses = "comments";
if ( loading ) { commentClasses += " loading-comments" }
```